README.md

```diff
@@ -21,7 +21,7 @@ This app is intended to be used on an offline computer by an election admin.
    - `election.json` - VotingWorks data format for election data: contests, canidates,ballot styles, precincts, etc.
    - approved official ballots in pdf format:
      - one for each combination of ballot style and precinct
-     - file name pattern: `election-${electionHash}-ballot-style-${ballotStyle}-precinct-id-${precinctId}.pdf`
+     - file name pattern: `election-${electionHash}-ballot-style-${ballotStyleId}-precinct-${dashify(precinctName)}-${precinctId}.pdf`
```

Comment:
Precinct name?

Reply:
I don't follow.

Reply:
Prefix of "id" was removed. 
--

Bigger question: are filenames for humans or computers, or both? 

My vote is for both. Short suggestion:

election-${electionId}-precinct-${dashify(precinct Name)}-id-${precinctId}-style-${ballotStyle}.pdf

Reply:
Alright, I changed it to what you suggested. Note that I interpreted `electionId` as a shorter version of `electionHash`.

Reply:
Sweet & sweet.